agent/pkg/up9/main.go

```diff
@@ -273,18 +292,33 @@ func syncEntriesImpl(token string, model string, envPrefix string, uploadInterva
 				logger.Log.Info("Stopping sync entries")
 				logger.Log.Fatal(postErr)
 			}
-			analyzeInformation.SentCount += len(entriesArray)
-			logger.Log.Infof("Finish uploading %v entries to %s\n", len(entriesArray), GetTrafficDumpUrl(envPrefix, model))
+			analyzeInformation.SentCount += 1
 
-			logger.Log.Infof("Uploaded %v entries until now", analyzeInformation.SentCount)
-		} else {
-			logger.Log.Infof("Nothing to upload")
+			if analyzeInformation.SentCount%SentCountLogInterval == 0 {
```

Comment:
Add spaces around `%`

Reply:
The [VS Code Go extension](https://marketplace.visualstudio.com/items?itemName=golang.Go) which is officially developed by Google and follow [Effective Go](https://golang.org/doc/effective_go) removes the spaces when I add them. Are you sure that's the correct coding style? Do we follow a different coding standard?